src/MarkdownEditor2022.csproj

```diff
@@ -1,157 +1,95 @@
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
+
   <ItemGroup>
-    <Compile Include="Commands\RefreshCommand.cs" />
-    <Compile Include="Commands\ShowMarkdownReferenceCommand.cs" />
-    <Compile Include="Commands\ShowKeybindingsCommand.cs" />
-    <Compile Include="Commands\OpenSettingsCommand.cs" />
-    <Compile Include="Commands\InsertLinkCommand.cs" />
-    <Compile Include="Commands\IndentationCommand.cs" />
-    <Compile Include="Commands\PasteImageCommand.cs" />
-    <Compile Include="Commands\ToggleTaskCommand.cs" />
-    <Compile Include="Commands\SmartIndentCommand.cs" />
-    <Compile Include="Commands\CommentCommand.cs" />
-    <Compile Include="Commands\EmphasizeTextCommand.cs" />
-    <Compile Include="Commands\UncommentCommand.cs" />
-    <Compile Include="Constants.cs" />
-    <Compile Include="Document.cs" />
-    <Compile Include="DragDrop\MarkdownDropHandler.cs" />
-    <Compile Include="DragDrop\MarkdownDropHandlerProvider.cs" />
-    <Compile Include="ExtensionMethods.cs" />
-    <Compile Include="Language\EditorFeatures.cs" />
-    <Compile Include="Language\MarkdownEditor.cs" />
-    <Compile Include="Language\MarkdownClassificationTypes.cs" />
-    <Compile Include="Language\TokenTagger.cs" />
-    <Compile Include="Margin\Browser.cs" />
-    <Compile Include="Margin\BrowserMargin.cs" />
-    <Compile Include="Margin\BrowserMarginProvider.cs" />
-    <Compile Include="Options\AdvancedOptions.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Commands\EnablePreviewSyncCommand.cs" />
-    <Compile Include="MarkdownEditor2022Package.cs" />
-    <Compile Include="source.extension.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>source.extension.vsixmanifest</DependentUpon>
-    </Compile>
-    <Compile Include="VSCommandTable.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>VSCommandTable.vsct</DependentUpon>
-    </Compile>
+    <AdditionalThisAssemblyFields Include="AssemblyDescription" String="$(Description)" />
   </ItemGroup>
+
   <ItemGroup>
-    <Content Include="Margin\highlight-dark.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\md-template.html">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\LICENSE">
-      <Link>Resources\LICENSE</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <None Include="source.extension.vsixmanifest">
-      <SubType>Designer</SubType>
-      <Generator>VsixManifestGenerator</Generator>
```

Comment:
📝 This shouldn't be necessary anymore (removed with no pending replacement)

Reply:
It needs the Generator to produce a .cs file used by the extension

Reply:
The .cs file has been replaced by code generated by Nerdbank.GitVersioning, and the generator is no longer needed.

Reply:
➡️ this change is moved to #9